### DISCUSSION TITLE
Windows Install Script Improvement

### DISCUSSION THREAD

**Discussion by jackdozier** [DigitalRuby/IPBan#299] [Ideas]
I've run into a case when installing on Windows machines where I don't want the service to automatically start before I've had a chance to update the config file. The RDP recipe has a default MaxFileSize setting that causes IPBan to remove older log files that are over that size limit. I've run into that limit in practice and it results in logs being deleted on install even though I plan to adjust the setting. There's no built-in way to prevent the install script from starting the service.

I was able to get around it by just downloading the script manually, commenting out the service start command, and running as normal. I also had the old logs in backups, so I was able to recover those easily.

An improvement, though, might be to add an optional parameter to the install script that allows you to suppress first start of the service. It could be helpful to include a message to the user reminding them to manually start the service after they have updated the config file.

Thanks for everything you do with IPBan!

**Comment by jjxtra** [comment]
Added in this commit: 32342115f3f564fc65f69218e2aa41f9ff0bcabc
